integrations/os2sync/os2mo.py

```diff
@@ -123,6 +125,9 @@ def addresses_to_user(user, addresses):
         if a["address_type"]["scope"] == "DAR":
             user["Location"] = a["name"]
 
+    phones = sorted(phones, key=lambda phone: phone["uuid"])
+    emails = sorted(emails, key=lambda email: email["uuid"])
```

Comment:
```suggestion
    from operator import itemgetter
    phones = sorted(phones, key=itemgetter('uuid'))
    emails = sorted(emails, key=itemgetter('uuid'))
```
Jeg ville nok bruge `itemgetter` her til at kommunikere at vi blot slår et givent felt op.